src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10565,12 +10558,9 @@ namespace UnitarySystems {
                         if (HeatSpeedNum == 0) {
                             state.dataUnitarySystems->CompOffMassFlow = this->MaxNoCoolHeatAirMassFlow;
                             state.dataUnitarySystems->CompOffFlowRatio = this->m_NoLoadAirFlowRateRatio;
-                        } else if (HeatSpeedNum == 1) {
+                        } else {
                             state.dataUnitarySystems->CompOffMassFlow = this->m_HeatMassFlowRate[HeatSpeedNum];
```

Comment:
Why doesn't this block set `CompOnMassFlow`? Throughout this function some block set both `CompOnMassFlow` and `CompOffMassFlow` but some just set the off values.

Reply:
Sorry, I did see this but got sidetracked and forgot to answer. Look at line 10536 above. This block of code is only setting the CompOffMassFlow based on whether or not the fan is cycling. The CompOnMassFlow gets set regardless of fan operation elsewhere (hopefully!).

Reply:
Ok, I see. There's a "default" `CompOnMassFlow` at the top of this section, and then some cases (maybe just 1) need to override that, but not all. I'll update this branch with develop and do a final review pass.